src/main/java/seedu/address/model/person/Person.java

```diff
@@ -27,13 +28,14 @@ public class Person {
     /**
      * Every field must be present and not null.
      */
-    public Person(Name name, Phone phone, Email email, Address address, Set<Tag> tags) {
+    public Person(Name name, Phone phone, Email email, Address address, GitHub gitHub, Set<Tag> tags) {
         requireAllNonNull(name, phone, email, address, tags);
```

Comment:
Should we make it so that phone, email, address and GitHub can accept a null input to better fit with our project?

If so, we will need to change line 39 in `AddCommandParser` to deal with null inputs too.
If we are making all fields mandatory, we should require the `github` field to be non-null. 
ie. `requireAllNonNull(name, phone, email, address, github, tags);`

Reply:
I removed the address component because I feel it doesn't align to the purpose. So currently, it is
Name, Phone, Email, Telegram, GitHub, Tag 

Alright thanks yeah I think we can make all of them non-null 👍

Reply:
Good catch thanks!